pkg/webui/components/navigation/side/item/item.styl

```diff
@@ -29,6 +30,15 @@
         height: auto
         justify-content: space-evenly
 
+      &:focus-within,
```

Comment:
I'm really not sure about this one and need help.
One of the requirements was to support accessing child elements via 'keyboard focus', so I did it with focus-within.
However adding this css property resulted in hover menu not closing after click.

Could anyone please suggest a proper solution to this issue or what am I missing?

Thanks!

Reply:
@bafonins could you please comment on this as well? Would this be the preferred solution on perhaps there's another way to do this? For example: autofocusing elements on page we get navigated to.